app/views/follows/_follow_button.html.erb

```diff
@@ -16,4 +16,4 @@
                   class: 'button hollow expanded' %>
     <% end %>
   </span>
-</div>
+</div>
```

Comment:
Please, include a trailing new line at the end of this file

Reply:
done